src/clients/http_client/__e2etests__/create_shopify_object.ts

```diff
@@ -0,0 +1,15 @@
+import {MemorySessionStorage} from '../../../session-storage/memory';
```

Comment:
Since this file is no longer creating the shopify object, merely the config, should the file be renamed?